### PR TITLE
Changelog back-log

### PR DESCRIPTION
**Pull request by emmanuelle**
I put here a few items corresponding to recent merged PRs, so that the changelog is more up to date. It's less painful to do this from time to time than during a release (ideally we should do it for each PR, but...)

One thing to discuss: I've copied links from my sprint blog post, so they have a markdown syntax. And we have this open PR #4863 to use mystify to be able to use markdown in the doc. Probably the change log would be nicer to write as markdown because of the numerous links. But existing release notes are in rst, so this might be a problem.

### CODE REVIEW

**Comment by sciunto** [comment]
Side comment: I think it is hard to ask contributors to add these entries and it is also difficult to take the extra time to make a new PR after each new feature. Don't you think a meta issue would be an easier way to collect those changes?

**Comment by emmanuelle** [comment]
@sciunto, yes it's hard to add this to the list of things to do for contributors, this is why I think a "catching-up" PR like this one can be a good solution from time to time. What did you have in mind for a "meta issue" ?

**Comment by sciunto** [comment]
More like we have a pinned issue in which we append comments or edit the first message to maintain the list.

**Comment by emmanuelle** [comment]
ok, but I think it's as much work to edit on github the `release_dev.rst` file and commit + PR the result, as editing a paragraph in an issue. Anyway, any workflow will work for me as long as the changelog is written gradually, as opposed as 90% remaining to be done as part of the release work.

**Comment by sciunto** [comment]
I'm not a super fan of the github interface for editing files. But if the majority prefers this solution, I'll follow :)

**Comment by emmanuelle** [comment]
> I'm not a super fan of the github interface for editing files. But if the majority prefers this solution, I'll follow :)

I don't use it much either, but to me editing an issue and editing a file with the github interface would not be very different :-).

File: doc/release/release_dev.rst [hunk 1]
```diff
@@ -15,11 +15,27 @@ https://scikit-image.org
...
 - A new doc tutorial presenting a cell biology example has been added to the
   gallery (#4648). The scientific content benefited from a much appreciated
   review by Pierre Poulain and Fred Bernard, both assistant professors at
   Université de Paris and Institut Jacques Monod.
+- New tutorial on [visualizing 3D data](https://scikit-image.org/docs/dev/auto_examples/applications/plot_3d_image_processing.html) ([#4850](https://github.com/scikit-image/scikit-image/pull/4850))
+- Documentation has been added to the contributing notes about how to submit a
+  gallery example 
+- automatic formatting of docstrings for improved consistency ([#4849](https://github.com/scikit-image/scikit-image/pull/4849))
+- improved docstring for `rgb2lab` ([#4839](https://github.com/scikit-image/scikit-image/pull/4839)) and `marching_cubes` [#4846](https://github.com/scikit-image/scikit-image/pull/4846)
+- improved docstring for `measure.marching_cubes`, mentioning how to decimate a
```
Comment (hmaarrfk) [inline doc/release/release_dev.rst:36]:
```suggestion
- Improved docstring for `measure.marching_cubes`, mentioning how to decimate a
```

**Comment by jni** [comment]
@emmanuelle I guess the failures have to do with the markdown syntax. Is your idea to have this as a long running PR? Or to wait until the MyST PR is merged? Or...?

**Comment by emmanuelle** [comment]
I'd prefer to wait until the myst PR is merged and then I'll either try to convert this file to md or transform the links to rst syntax.

**Comment by jni** [comment]
@emmanuelle I've marked this as draft so no one is tempted to merge before that happens, then. =)

**Comment by sciunto** [comment]
@jni how did you do that? I never found the github doc for that.

**Comment by jni** [comment]
@sciunto 

Screen Shot 2020-08-16 at 5 22 35 pm

It's subtle but that's actually a thing you can click on! UI fail? 🤷

**Comment by sciunto** [comment]
I never noticed! thank you!

**Comment by emmanuelle** [comment]
@scikit-image/core I revived this old PR, which will be useful for the release.

**Comment by hmaarrfk** [comment]
Thanks for the heroic effort. Looking pretty good!

**Comment by stefanv** [comment]
I wonder if we should consider using something like Towncrier?  Not important for now, just wondered if people liked the way, e.g., NumPy is doing is.

**Comment by hmaarrfk** [comment]
I can help set that up. But for now, shall we merge before it gets too stale ;)

File: doc/release/release_dev.rst [hunk 2]
```diff
@@ -30,27 +64,49 @@ Improvements
...
 - ``pyproject.toml`` has been added to the sdist.
-
 - The performance of the SLIC superpixels algorithm
   (``skimage.segmentation.slice``) was improved for the case where a mask
   is supplied by the user (#4903). The specific superpixels produced by
   masked SLIC will not be identical to those produced by prior releases.
+- ``exposure.adjust_gamma`` has been accelerated for ``uint8`` images thanks to a
+  LUT (#4966).  
+- ``measure.label`` has been accelerated for boolean input images, by using
+  ``ndimage``'s implementation for this case (#4945).
```
Comment (grlee77) [inline doc/release/release_dev.rst:74]:
```suggestion
  ``scipy.ndimage``'s implementation for this case (#4945).
```

nitpick: explicitly mention scipy for clarity

File: doc/release/release_dev.rst [hunk 3]
```diff
@@ -30,27 +64,49 @@ Improvements
...
+- ``data.cat`` has been introduced as an alias of ``data.chelsea`` for a more
+  explicit name.
+- The ``level`` parameter of ``measure.find_contours`` is now a keyword
+  argument, with a default value set to (max(image) - min(image)) / 2.
 
 
 Bugfixes
 --------
 
+- For the ransac algorithm, improved the case where all data points are
```
Comment (grlee77) [inline doc/release/release_dev.rst:101]:
```suggestion
- For the RANSAC algorithm, improved the case where all data points are 
```

**Review by grlee77 — APPROVED** [review]
looks good to me, I just left 2 minor suggestions

File: doc/release/release_dev.rst [hunk 4]
```diff
@@ -15,13 +15,47 @@ https://scikit-image.org
 New Features
 ------------
 
+- A new function ``segmentation.expand_labels`` has been added in order to dilate
+  labels without overlap #4795
```
Comment (jni) [inline doc/release/release_dev.rst:19]:
```suggestion
  labels while preventing overlap (#4795)
```

File: doc/release/release_dev.rst [hunk 5]
```diff
@@ -15,13 +15,47 @@ https://scikit-image.org
...
+- It is now possible to pass extra measurement functions to
+  ``measure.regionprops`` and ``regionprops_table`` #4810
```
Comment (jni) [inline doc/release/release_dev.rst:21]:
```suggestion
  ``measure.regionprops`` and ``regionprops_table`` (#4810)
```

File: doc/release/release_dev.rst [hunk 6]
```diff
@@ -15,13 +15,47 @@ https://scikit-image.org
...
 
 - A new doc tutorial presenting a cell biology example has been added to the
   gallery (#4648). The scientific content benefited from a much appreciated
   review by Pierre Poulain and Fred Bernard, both assistant professors at
   Université de Paris and Institut Jacques Monod.
+- New tutorial on `visualizing 3D data <https://scikit-image.org/docs/dev/auto_examples/applications/plot_3d_image_processing.html>`_ (#4850)
+- Documentation has been added to the contributing notes about how to submit a
+  gallery example 
+- automatic formatting of docstrings for improved consistency (#4849)
+- improved docstring for ``rgb2lab`` (#4839) and ``marching_cubes`` (#4846)
```
Comment (jni) [inline doc/release/release_dev.rst:41]:
```suggestion
- Automatic formatting of docstrings for improved consistency (#4849)
- Improved docstring for ``rgb2lab`` (#4839) and ``marching_cubes`` (#4846)
```

(or change everything else to lowercase... ;)

File: doc/release/release_dev.rst [hunk 7]
```diff
@@ -15,13 +15,47 @@ https://scikit-image.org
...
+- Improved docstring for ``measure.marching_cubes``, mentioning how to decimate a
+  mesh using mayavi #4846
```
Comment (jni) [inline doc/release/release_dev.rst:43]:
```suggestion
  mesh using mayavi (#4846)
```

File: doc/release/release_dev.rst [hunk 8]
```diff
@@ -30,27 +64,49 @@ Improvements
...
+  #4862
 - ``skimage.restoration.richardson_lucy`` returns a single-precision output
   when the input is single-precision. Prior to this release, double-precision
   was always used.
 - The default value of ``threshold_rel`` in ``skimage.feature.corner`` has
   changed from 0.1 to None, which corresponds to letting 
   ``skimage.feature.peak_local_max`` decide on the default. This is currently
   equivalent to ``threshold_rel=0``.
+- ``data.cat`` has been introduced as an alias of ``data.chelsea`` for a more
+  explicit name.
```
Comment (jni) [inline doc/release/release_dev.rst:93]:
```suggestion
  descriptive name.
```

File: doc/release/release_dev.rst [hunk 9]
```diff
@@ -30,27 +64,49 @@ Improvements
...
+- The ``level`` parameter of ``measure.find_contours`` is now a keyword
+  argument, with a default value set to (max(image) - min(image)) / 2.
 
 
 Bugfixes
 --------
 
+- For the ransac algorithm, improved the case where all data points are 
+  outliers, which was previously raising an error 
+  #4844
```
Comment (jni) [inline doc/release/release_dev.rst:103]:
```suggestion
  (#4844)
```

File: doc/release/release_dev.rst [hunk 10]
```diff
@@ -30,27 +64,49 @@ Improvements
...
+- An error-causing bug has been corrected for the ``bg_color`` parameter in
+  ``label2rgb`` when its value is a string #4840
+- A normalization bug was fixed in ``metrics.variation_of_information`` 
+  #4875
+- Fixed the behaviour of Richardson-Lucy deconvolution for images with 3
+  dimensions or more #4823
```
Comment (jni) [inline doc/release/release_dev.rst:109]:
```suggestion
  ``label2rgb`` when its value is a string (#4840)
- A normalization bug was fixed in ``metrics.variation_of_information`` 
  (#4875)
- Fixed the behaviour of Richardson-Lucy deconvolution for images with 3
  dimensions or more (#4823)
```

File: doc/release/release_dev.rst [hunk 11]
```diff
@@ -59,7 +115,16 @@ Bugfixes
   They have been replaced with ``nrow`` and ``ncol``.
 - ``skimage.segmentation.flood_fill`` and ``skimage.segmentation.flood``
   now consistently handle negative values for ``seed_point``.
-- In `skimage.draw.polygon`, segmentation fault caused by 0d inputs.
+- In ``skimage.draw.polygon``, a segmentation fault caused by 0d inputs has
+  been fixed.
```
Comment (jni) [inline doc/release/release_dev.rst:119]:
```suggestion
```

(this is repeated below)

**Review by jni — APPROVED** [review]
Awesome. This is going to be a great release. =D

**Comment by jni** [comment]
(Sorry about my formatting OCD 😅)

**Comment by hmaarrfk** [comment]
> (Sorry about my formatting OCD 😅)

Is it still policy to suggest these kinds of changes instead of just pushing them?

**Comment by jni** [comment]
> Is it still policy to suggest these kinds of changes instead of just pushing them?

Sorry about that. I was on the GitHub UI and it doesn't (afaict?) let me just push. For a new contributor I would definitely not have been this nitpicky, but I trust @emmanuelle can quickly do the "add suggestion to batch" pass and commit from the web UI — even more quickly than me checking out the PR. If that assessment was wrong, I definitely apologise!

**Comment by hmaarrfk** [comment]
Right, I've just been editing the files directly in a core members branch. I just wanted make sure that was ok since I find that the back and forth for these kinds of PRs can be really slow.

**Comment by hmaarrfk** [comment]
i was mosrly making sure that what i do is ok.

**Comment by emmanuelle** [comment]
@hmaarrfk yes pushing these commits to a contributor's branch is perfectly fine :-). I've accepted all changes, so given the fact that there are three +1, I think the PR can be merged! Thanks for the improvements.

**Pull request merged**